.github/workflows/conda-python-tests.yaml

```diff
@@ -113,10 +111,12 @@ jobs:
       - name: Generate test report
         uses: test-summary/action@v2
         with:
-          paths: "${{ inputs.tests_result_dir }}/*.xml"
+          paths: "${{ env.RAPIDS_TESTS_DIR }}/*.xml"
         if: always()
-      - uses: codecov/codecov-action@v3
-        if: inputs.run_codecov && inputs.build_type == 'pull-request'
-        with:
-          directory: ${{ env.RAPIDS_COVERAGE_DIR }}
-          verbose: true
+      - name: Run codecov
+        if: inputs.run_codecov && contains(fromJson('["X64", "X86"]'), runner.arch)
```

Comment:
We probably only need one value here, right? Which is it?

Reply:
Probably only `X64`. I got the full list of values at the link below. I'll update it.

- https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context